Examples/Algorithms/Digitization/include/ActsExamples/Digitization/SmearingAlgorithm.hpp

```diff
@@ -8,15 +8,11 @@
 
 #pragma once
 
-#include "Acts/EventData/Measurement.hpp"
-#include "ActsExamples/EventData/DigitizedHit.hpp"
-#include "ActsExamples/EventData/GeometryContainers.hpp"
+#include "Acts/Geometry/GeometryHierarchyMap.hpp"
+#include "Acts/Utilities/ParameterDefinitions.hpp"
```

Comment:
What's the policy on this?

If it's part of Acts/ActsFatras/ActsExamples it's done with quotes, otherwise with brackets?

Reply:
Yes, `#include "..."` for local headers, i.e. those that are part of the project, and `#include <...>` for everything else, i.e. and third party libraries and external dependencies. I am not sure if this is written down in the guidelines, but it is quite consistently followed in the code base.